src/App.css

```diff
@@ -1,14 +1,14 @@
-.App {
-  text-align: center;
-}
-
 body {
   display: flex;
```

Comment:
- [ ] You have used `display: flex` 7 times, which falls under the category of bad practice since you are repeating yourself. I would suggest you create a class with display: flex and give it to any html element that you need